src/controllers/controller.line.js

```diff
@@ -283,9 +283,15 @@ module.exports = function(Chart) {
 			var points = meta.data || [];
 			var area = chart.chartArea;
 			var ilen = points.length;
+			var halfBorderWidth = (meta.dataset._model.borderWidth || 0) / 2;
```

Comment:
I should have read `update()` more carefully because `_model` is only available if `options.showLine: true`.

We could also avoid clipping if there is nothing to draw ...

```javascript
if (lineEnabled(me.getDataset(), chart.options)) {
    halfBorderWidth = (meta.dataset._model.borderWidth || 0) / 2;

    helpers.canvas.clipArea(chart.ctx, {
        left: area.left,
        right: area.right,
        top: area.top - halfBorderWidth,
        bottom: area.bottom + halfBorderWidth
    });

    meta.dataset.draw();

    helpers.canvas.unclipArea(chart.ctx);
}
```

Reply:
Please verify that it works locally for your, I didn't test it, neither that tests pass (`gulp test`).